content/docs/alerting/configuration.md

```diff
@@ -64,10 +64,16 @@ global:
   # The default SMTP From header field.
   [ smtp_from: <tmpl_string> ]
   # The default SMTP smarthost used for sending emails.
+  # Provide the correct port for the smarthost:
+  # 25 for servers not providing TLS / SSL (set smtp_require_tls to "false" 
+  # 587 for STARTTLS servers
```

Comment:
I don't believe that's the name of the protocol.

Reply:
STARTTLS is the name of the SMTP command: https://en.wikipedia.org/wiki/Opportunistic_TLS

Reply:
So it's not the name of the protocol in question, and thus not the name of the port

Reply:
I changed the explanation by a bit to make this distinction clear